src/Nova/Contact.php

```diff
@@ -83,9 +83,9 @@ protected function submissionFields()
     {
         return [
             Text::make('Number', 'reference_number')->exceptOnForms(),
-            Number::make('Participants')->nullable()->hideWhenUpdating(),
-            Date::make('Requested Date')->nullable()->hideWhenUpdating(),
-            Text::make('Requested Time')->nullable()->hideWhenUpdating(),
+            Number::make('Participants', 'fields->participants')->nullable()->hideWhenUpdating(),
```

Comment:
@pdbreen for accessing values in a json column in Nova, is adding `fields` as a protected model cast then calling the values this way how it should be done? Or is there another way this should be handled? For reference, `fields` is a json column that can be found in the create contacts migration.

Reply:
Let's just use "form_fields" for this. That will solve the issue

Reply:
The answer really depends on the UI that you want to expose in Nova.  I saw in a later commit you went with the Nova json field type, which is fine - but may be difficult for staff to fill in properly.  If that becomes true and there is a desire for a more data-type aware interface, the simplest next step would be to add get/set attribute methods on the model and have those methods pull/push data into the actual json data field.  Then, in Nova, you can use the calculated attribute as the data source for a Nova field.

Reply:
Thank you, Patrick. I think with this package, the staff just needs to be able to read the various fields. They won't have the ability to edit the information submitted via the contact forms.